src/main/preload/mattermost.js

```diff
@@ -152,11 +152,11 @@ window.addEventListener('message', ({origin, data = {}} = {}) => {
         }
         break;
     }
-    case 'browser-history-push': {
-        const {path} = message;
-        ipcRenderer.send(BROWSER_HISTORY_PUSH, viewName, path);
-        break;
-    }
+    // case 'browser-history-push': {
```

Comment:
(nitpick) this(same for below (line 286+)) can be deleted instead of commented I guess (we still have the repo history if we want to bring it back).